### ISSUE TITLE
Pupil Player Video Export 1.2-7

### ISSUE THREAD

**Issue by fneitzel** [pupil-labs/pupil#1050]
```
2018-01-31 17:02:34,276 - MainProcess - [INFO] os_utils: Disabling idle sleep not supported on this OS version.
2018-01-31 17:02:34,751 - player - [INFO] launchables.player: Session setting are from a  different version of this app. I will not use those.
2018-01-31 17:02:58,718 - player - [INFO] launchables.player: Starting new session with 'D:\MTM_export\002'
2018-01-31 17:02:58,721 - player - [INFO] player_methods: Updating meta info
2018-01-31 17:02:59,169 - player - [INFO] video_capture: Install pyrealsense to use the Intel RealSense backend
2018-01-31 17:02:59,327 - player - [INFO] launchables.player: Application Version: 1.2.7
2018-01-31 17:02:59,327 - player - [INFO] launchables.player: System Info: User: mattern, Platform: Windows, Machine: leibovitz, Release: 10, Version: 10.0.16299
2018-01-31 17:02:59,551 - player - [INFO] camera_models: Previously recorded calibration found and loaded!
2018-01-31 17:02:59,551 - player - [INFO] launchables.player: Session setting are a different version of this app. I will not use those.
2018-01-31 17:06:11,712 - player - [WARNING] launchables.player: Previous export for range [0-74221] already exsits - overwriting.
2018-01-31 17:06:11,715 - player - [WARNING] video_export_launcher: Adding new video export process.
2018-01-31 17:07:51,802 - player - [ERROR] launchables.player: Process Player crashed with trace:
Traceback (most recent call last):
  File "launchables\player.py", line 415, in player
  File "shared_modules\video_export_launcher.py", line 97, in on_notify
  File "shared_modules\video_export_launcher.py", line 120, in add_export
  File "shared_modules\background_helper.py", line 37, in __init__
  File "multiprocessing\process.py", line 105, in start
  File "multiprocessing\context.py", line 223, in _Popen
  File "multiprocessing\context.py", line 322, in _Popen
  File "multiprocessing\popen_spawn_win32.py", line 65, in __init__
  File "multiprocessing\reduction.py", line 60, in dump
OSError: [Errno 22] Invalid argument

2018-01-31 17:07:51,812 - player - [INFO] launchables.player: Process shutting down.
```

The video is just exportable with my 16GB RAM PC on Ubuntu. On any other laptop it will crash due to: issue #889 

Best
Freddy

**Comment by willpatera** [comment]
@fneitzel we are building the `v1.3` Windows bundles now. I will try to test/replicate this behavior before relasing the bundle.

**Comment by willpatera** [comment]
I am closing this issue as issues with large recordings will be addressed with #1141.
